app/assets/javascripts/discourse/app/components/notification-consent-banner.js

```diff
@@ -20,21 +20,19 @@ export default DesktopNotificationConfig.extend({
     "isNotSupported",
     "isEnabled",
     "bannerDismissed",
-    "currentUser.reply_count",
-    "currentUser.topic_count"
+    "currentUser.any_posts"
   )
   showNotificationPromptBanner(
     isNotSupported,
     isEnabled,
     bannerDismissed,
-    replyCount,
-    topicCount
+    any_posts
```

Comment:
We use `camelCase` on the client-side (except very few places) 😉 

```suggestion
    anyPosts
```